### DISCUSSION TITLE
Classification Preview

### DISCUSSION THREAD

**Discussion by Almeida100** [semiautomaticgit/SemiAutomaticClassificationPlugin#308] [Q&A] [answered]
I'm using a Sentinel2-Lvel2A band set, clipped to my Area Of Interest.
When I use Classification Preview tool, I get a sample with a pixel size of 60 meters.
How can I change the size of this pixel to 10 meters, so that every time I use Classification Preview, the sample pixel has this size?

**Comment by semiautomaticgit** [comment]
Hello @Almeida100 ,
probably you have included 60m bands in the Band set. You should exclude these bands from the band set to get 10m output.
Alternatively, you should move a 10m band to the first band of the band set, or resample all the input bands to 10m using QGIS tools.

**Reply by Almeida100** [reply, marked as answer]
Thank you very much for your quick response.
I carried out the work in this order:
1 - I cut the JP2 bands according to my area of interest;
2 - I suppressed bands B01 and B09 and converted the remaining bands to TIF rasters; the 20 m pixel size bands were automatically converted to 10 m pixels.
I think that the problem of the existence of singular covariance matrices can be avoided by creating some polygonal ROIs (instaed using only Region Growing Algorithm).

**Reply by semiautomaticgit** [reply]
You are very welcome! 

> I think that the problem of the existence of singular covariance matrices can be avoided by creating some polygonal ROIs (instaed using only Region Growing Algorithm).

Yes, larger ROIs can solve the issue
